app.js

```diff
@@ -53,13 +52,58 @@ function googleAnalyticsMiddleware(data) {
     }
 }
 
+function forceUpgrade(html) {
+    var meta = [
+		"<meta http-equiv=\"Content-Security-Policy\" content=\"upgrade-insecure-requests\">"
+		].join("\n");
+    html = html.replace("</head>", meta + "\n\n</head>");
+    return html;
+}
+
+function forceHttpsUpgradeMiddleware(data) {
+    if (data.contentType == 'text/html') {
+
+        // https://nodejs.org/api/stream.html#stream_transform
+        data.stream = data.stream.pipe(new Transform({
+            decodeStrings: false,
+            transform: function(chunk, encoding, next) {
+                this.push(forceUpgrade(chunk.toString()));
+                next();
+            }
+        }));
+    }
+}
+
+function tetrioPatch(html) {
+    var meta = [
+		"<meta http-equiv=\"Content-Security-Policy\" content=\"upgrade-insecure-requests\">"
+		].join("\n");
+    html = html.replace("<meta name=googlebot content=notranslate>", "<meta name=googlebot content=notranslate>\n\n" + meta);
+    return html;
+}
+
+function tetrioPatchMiddleware(data) {
+    if (data.contentType == 'text/html') {
+
+        // https://nodejs.org/api/stream.html#stream_transform
+        data.stream = data.stream.pipe(new Transform({
+            decodeStrings: false,
+            transform: function(chunk, encoding, next) {
+                this.push(tetrioPatch(chunk.toString()));
+                next();
+            }
+        }));
+    }
+}
```

Comment:
Is the basic issue that tetr.io requires https, and so this is one way of forcing https?

What does this do if you only have http, and don't have a https server?

Also, if it's specifically for tetr.io, then the if check should be conditioned on that in addition to the contentType, similar to how we check for youtube.com here: https://github.com/nfriedly/node-unblocker/blob/ee8358df24dc6abe2cc4884c48e3d0e44b57c0b2/examples/youtube/youtube.js#L7-L8

Lastly, if we do keep this, I think this should go in a separate file that's just imported into app.js.